build-image/first-boot.bash

```diff
@@ -46,20 +46,20 @@ if is_pi; then
 fi
 
 echo -n "$(timestamp) [openHABian] Changing default username and password... "
-if [ -z ${username+x} ] || ! id $userdef &>/dev/null || id "$username" &>/dev/null; then
+if [ -z "${username:-openhabian}" ] || ! id $userdef &>/dev/null || id "$username" &>/dev/null; then
```

Comment:
This change will break this routine. Have a look at: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02
Insert exception `# shellcheck disable=SC2154`

Reply:
ok